napari/utils/_tests/test_io.py

```diff
@@ -111,6 +111,13 @@ def test_single_filename(single_tiff):
     assert images.shape == (2, 15, 10)
 
 
+def test_guess_zarr_path():
+    assert io.guess_zarr_path('dataset.zarr')
+    assert io.guess_zarr_path('dataset.zarr/some/long/path')
+    assert not io.guess_zarr_path('data.tif')
+    assert not io.guess_zarr_path('no_zarr_suffix/data.png')
```

Comment:
post review comment: it would be good to test the behaviour on `'path/to/image.zarr/component.png'`. That will return True currently and I think that's the correct behaviour, but one would have to think about it, it would be great to have it be "documented". Maybe this test belongs in a doctest in the docstring for the function.

Reply:
I added a doctest, but left the unit test with more edge cases that hopefully aren't up for debate over what `guess_zarr_path` should return. For the case you mentioned, I think it's the correct behavior as well, since having `.zarr` _somewhere_ in the path should indicate directories below are groups or arrays (regardless of the component name).